src/openassetio-core/include/openassetio/pluginSystem/HybridPluginSystemManagerImplementationFactory.hpp

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2024 The Foundry Visionmongers Ltd
+#pragma once
+#include <vector>
+
+#include <openassetio/export.h>
+#include <openassetio/hostApi/ManagerImplementationFactoryInterface.hpp>
+#include <openassetio/typedefs.hpp>
+
+OPENASSETIO_FWD_DECLARE(managerApi, ManagerInterface)
+
+namespace openassetio {
+inline namespace OPENASSETIO_CORE_ABI_VERSION {
+namespace pluginSystem {
+OPENASSETIO_DECLARE_PTR(HybridPluginSystemManagerImplementationFactory)
+
+/**
+ * The hybrid plugin system composes one or more child plugin systems,
+ * and abstracts away routing API calls based on priority and
+ * capability.
+ *
+ * A list of factories are provided in priority order. When a plugin
+ * with a particular identifier is requested, all factories are queried
+ * and any that return positively for the identifier have their
```

Comment:
Oh yeah you should provide all available plugins to OpenAssetIO. A host application is free to support multiple manager plugins - though the simplest workflow, using a global `.toml` config file, only supports one at a time.  

It would be an advanced use-case for host applications to support, but we've had people in the TSC meeting saying that they use multiple manager-like systems in their studio.

I guess a more immediate use-case would be upgrading from one system to another (e.g. Shotgrid to in-house), where you can have both plugins available, then its a matter of toggling the config to switch some/all machines between backends.

Reply:
I'm unsure whether anything should come of this conversation. Perhaps some documentation indicating the use cases described?

Reply:
Mentioned this as part of docs update in 448cadfc